app/views/notes/_form.html.erb

```diff
@@ -2,32 +2,39 @@
 
 <div>
   <%= form_for(note) do |f| %>
-    <div>
+    <div class="field">
       <label for="name">Name: </label>
       <%= f.text_area :name, placeholder: "meeting_notes_2016-08-04", class: 'form-field' %>
     </div>
 
-    <div>
+    <div class="field">
       <label for="body">Body: </label>
       <%= f.text_area :body, placeholder: "Today we started planning...", class: 'form-field' %>
     </div>
 
-    <div>
+    <div class="field">
       <label for="file_type">File Extension: </label>
       <%= f.select :file_type, options_from_collection_for_select(["txt", "md"], :file_type) %>
     </div>
 
-    <div>
+    <div class="field">
       <label for="category_id">Category: </label>
       <%= f.select :category_id, options_from_collection_for_select(Category.all, :id, :name) %>
     </div>
 
+
+    <div class="field">
```

Comment:
Make sure you don't have empty divs hanging out, and that they're formatted correctly (less spacing)